pyttsx3/drivers/avsynth.py

```diff
@@ -0,0 +1,129 @@
+import objc
+from AppKit import NSSpeechSynthesizer
+from Foundation import *
```

Comment:
Let's not do wildcard imports as discussed in PEP8 because the blind liinters.

Reply:
Sure. It's a good point. I'm being lazy

I think this is all of them

`from Foundation import NSURL, NSTimer, NSRunLoop, NSDefaultRunLoopMode, NSDate`